src/seedu/addressbook/data/person/Address.java

```diff
@@ -38,19 +47,19 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return value;
+        return blockValue.value + "," + streetValue.value + "," + unitValue.value + "," + postalCodeValue.value;
```

Comment:
Is there a better way of accessing the values? Recall examples on encapsulation during tutorial.